src/backend/distributed/transaction/transaction_management.c

```diff
@@ -185,6 +193,64 @@ InCoordinatedTransaction(void)
 }
 
 
+/*
+ * Sets a flag to true indicating that the current node is executing a delegated
+ * function call, using forcePushdown, within a distributed transaction issued
+ * by the coordinator. Also, saves the distribution argument.
+ */
+void
+EnableInForceDelegatedFuncExecution(Const *distArgument)
+{
+	MemoryContext oldcontext = MemoryContextSwitchTo(TopTransactionContext);
+	elog(DEBUG1, "Saving Distribution Argument: %s",
+		 pretty_format_node_dump(nodeToString(distArgument)));
```

Comment:
should use ereport to avoid very expensive deparsing

Reply:
Done